easypy/misc.py

```diff
@@ -40,8 +40,10 @@ def stack_level_to_get_out_of_file():
     return stack_levels
 
 
-def clamp(val, *, at_least=None, at_most=None):
-    "Clamp a value so it doesn't exceed specified limits"
+def clamp(val, at_least=None, at_most=None):
```

Comment:
no - as discussed, `clamp` should require all 3 params (though they can be passed `None`; which should be documented).